integ-test-promote.pipeline.yml

```diff
@@ -1,485 +1,560 @@
-resources:
```

Comment:
I don't understand why it shows every line of this file changing when many lines have not changed.

Reply:
I'm not too sure either. Maybe something fishy with line-endings? I can take a look

Reply:
I think this might just be a GitHub thing. If the file is sufficiently long and enough lines have changed, they just consider the whole file changed.